doc/user/content/self-hosted/install-on-local-kind.md

```diff
@@ -0,0 +1,246 @@
+      service/mzfhj38ptdjs-cluster-s2-replica-s2-gen-1   ClusterIP   10.96.120.116   <none>        2100/TCP,2103/TCP,2101/TCP,2102/TCP,6878/TCP                                                3m25s
+      service/mzfhj38ptdjs-cluster-s3-replica-s3-gen-1   ClusterIP   10.96.187.199   <none>        2100/TCP,2103/TCP,2101/TCP,2102/TCP,6878/TCP                                                3m25s
+      service/mzfhj38ptdjs-cluster-u1-replica-u1-gen-1   ClusterIP   10.96.92.133    <none>        2100/TCP,2103/TCP,2101/TCP,2102/TCP,6878/TCP                                                3m25s
+      service/mzfhj38ptdjs-console                       NodePort    10.96.97.5      <none>        9000:30847/TCP                                                                              3m21s
+      service/mzfhj38ptdjs-environmentd                  NodePort    10.96.188.140   <none>        6875:30525/TCP,6876:31052/TCP,6877:31711/TCP,6878:31367/TCP,6880:30141/TCP,6881:30283/TCP   3m21s
+      service/mzfhj38ptdjs-environmentd-1                NodePort    10.96.228.68    <none>        6875:32253/TCP,6876:31876/TCP,6877:31886/TCP,6878:31643/TCP,6880:32409/TCP,6881:30932/TCP   3m29s
+      service/mzfhj38ptdjs-persist-pubsub-1              ClusterIP   None            <none>        6879/TCP                                                                                    3m29s
+
+      NAME                                     READY   UP-TO-DATE   AVAILABLE   AGE
+      deployment.apps/mzfhj38ptdjs-balancerd   1/1     1            1           3m21s
+      deployment.apps/mzfhj38ptdjs-console     2/2     2            2           3m21s
+
+      NAME                                                DESIRED   CURRENT   READY   AGE
+      replicaset.apps/mzfhj38ptdjs-balancerd-6dd5bb645d   1         1         1       3m21s
+      replicaset.apps/mzfhj38ptdjs-console-84cb5c98d6     2         2         2       3m21s
+  
+      NAME                                                      READY   AGE
+      statefulset.apps/mzfhj38ptdjs-cluster-s1-replica-s1-gen-1   1/1     3m25s
+      statefulset.apps/mzfhj38ptdjs-cluster-s2-replica-s2-gen-1   1/1     3m25s
+      statefulset.apps/mzfhj38ptdjs-cluster-s3-replica-s3-gen-1   1/1     3m25s
+      statefulset.apps/mzfhj38ptdjs-cluster-u1-replica-u1-gen-1   1/1     3m25s
+      statefulset.apps/mzfhj38ptdjs-environmentd-1                1/1     3m29s
+      ```
+
+      If you run into an error during deployment, refer to the
+      [Troubleshooting](/self-hosted/troubleshooting) guide.
+
+1. Open the Materialize console in your browser:
+
+   1. From the `kubectl` output, find the Materialize console service.
+
+      ```shell
+      NAME                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S) 
+      service/mzfhj38ptdjs-console   NodePort    10.96.97.5      <none>        9000:30847/TCP
+      ```
+
+   1. Forward the Materialize console service to your local machine:
```

Comment:
I had to forward on my mac.

Reply:
We discussed this some more on the Cloud team, and likely will keep forwarding as the solution here, and generally let users decide what ingress strategy they want